.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - macos-latest
+          - ubuntu-latest
+        target:
+          - 0.20.2
+          - 1.0.10
+          - 1.2.8
+          - 1.4.0
+          - git:8a004e2
```

Comment:
is this just a demo? :) 

Pretty cool, but we don't need this here.

Reply:
This mimics the current Travis build: https://github.com/nim-lang/nimble/blob/master/.travis.yml#L13 I'm fine removing it.

Reply:
well, it's either this or the nightly. We don't need both. If the nightly works and is newer let's keep it instead of this.